examples/subscriptions/tsconfig.json

```diff
@@ -1,23 +1,11 @@
 {
+  "extends": "../../templates/skeleton/tsconfig.json",
   "include": ["./**/*.d.ts", "./**/*.ts", "./**/*.tsx"],
```

Comment:
This removes the need for a local `env.d.ts` (we do the same in the inifinite-scroll example):

```suggestion
  "include": [
    "./**/*.d.ts",
    "./**/*.ts",
    "./**/*.tsx",
    "../../templates/skeleton/*.d.ts"
  ],
```